omas/machine_mappings/mast.py

```diff
@@ -0,0 +1,322 @@
+
+def get_pyuda_client(server=None, port=None):
+    if server is not None:
+        pyuda.Client.server = server
+    elif not os.environ['UDA_HOST']:
+        raise pyuda.UDAException('Must set UDA_HOST environmental variable')
+
+    if port is not None:
+        pyuda.Client.port = port
+    elif not os.environ['UDA_PORT']:
+        raise pyuda.UDAException('Must set UDA_PORT environmental variable')
+
+    return pyuda.Client()
+
+def nstx_filenames(filename, pulse):
+
+    return support_filenames('mast', filename, pulse)
+
+
+
+@machine_mapping_function(__regression_arguments__, pulse=44653)
+def pf_active_hardware(ods, pulse):
+    r"""
+    Loads MAST tokamak poloidal field coil hardware geometry
+
+    :param ods: ODS instance
+    """
+    from omfit_classes.omfit_efund import OMFITmhdin, OMFITnstxMHD
+
+    mhdin = get_support_file(OMFITmhdin, nstx_filenames('mhdin', pulse))
+    mhdin.to_omas(ods, update=['oh', 'pf_active'])
+
+    signals = get_support_file(OMFITnstxMHD, nstx_filenames('signals', pulse))
+    icoil_signals = signals['mappings']['icoil']
+    oh_signals = signals['mappings']['ioh']
+
+    c_pf = 0
+    c_oh = 0
+    for c in ods[f'pf_active.coil']:
+        if 'OH' in ods[f'pf_active.coil'][c]['name']:
+            c_oh += 1
+            cname = oh_signals[c_oh]['name']
+            if oh_signals[c_oh]['mds_name'] is None:
+                cid = 'None'
+            else:
+                cid = oh_signals[c_oh]['mds_name'].strip('\\').replace("~", " ")
+        else:
+            c_pf += 1
+            cname = icoil_signals[c_pf]['name']
+            if icoil_signals[c_pf]['mds_name'] is None:
+                cid = 'None'
+            else:
+                cid = icoil_signals[c_pf]['mds_name'].strip('\\').replace("~", " ")
+        for e in ods[f'pf_active.coil'][c]['element']:
+            # if 'OH' in ods[f'pf_active.coil'][c]['name']:
+            #    ename = oh_signals[c_oh]['mds_name_resolved'].strip('\\') + f'_element_{e}'
+            # else:
+            #    ename = icoil_signals[c_pf]['mds_name_resolved'].strip('\\') + f'_element_{e}'
+            ename = cid + f'_element_{e}'
+            eid = ename
+            #ods[f'pf_active.coil'][c]['name'] = cname
+            ods[f'pf_active.coil'][c]['identifier'] = cid
+            ods[f'pf_active.coil'][c]['element'][e]['name'] = ename
+            ods[f'pf_active.coil'][c]['element'][e]['identifier'] = eid
+
+
+@machine_mapping_function(__regression_arguments__, pulse=44653)
+def pf_active_coil_current_data(ods, pulse, server=None, port=None):
+    r"""
+    Load MAST tokamak pf_active coil current data
+
+    :param ods: ODS instance
+
+    :param pulse: shot number
+    """
+    from omfit_classes.omfit_efund import OMFITnstxMHD
+    from omfit_classes.utils_math import firFilter
+    ods1 = ODS()
+    unwrap(pf_active_hardware)(ods1, pulse)
+
+    signals = get_support_file(OMFITnstxMHD, nstx_filenames('signals', pulse))
```

Comment:
`OMFITnstxMHD` and `nstx_filenames` in `mast.py`?

Reply:
changed nstx_filenames to mast_filenames